content/en/lotus/manage/chain-management.md

```diff
@@ -65,15 +52,6 @@ These lightweight state snapshots **do not contain any message receipts**. To ge
     lotus daemon --import-snapshot minimal_finality_stateroots_517061_2021-02-20_11-00-00.car
```

Comment:
This is not the filename that the new service produces, it is something like `2262960_2022_10_19T16_00_00Z.car` instead ( no `minimal_finality_stateroots` prefix.

cc @travisperson in case this was not intentional

Reply:
Theres no `minimal_finality_stateroots` at the new snapshots.

Reply:
@Reiers right... then this needs fixing, something like:
```suggestion
    lotus daemon --import-snapshot 2262960_2022_10_19T16_00_00Z.car
```